### PR TITLE
Add support for Python3.9

### PR DESCRIPTION
**Pull request by JAVGan**
# Add support for Python 3.9

This PR is a subsequent implementation of #355 and should be only merged **after** it.

It includes the support for Python3.9 as well as keep Python3.8 as default for IIB.

## Black update issues
The code-formatter `black` just support Python 3.9 from version [21.4b0](https://pypi.org/project/black/21.4b0/) and above. 

However, after upgrading `black` from `19.10b0` to `21.4b0` it complained about the formatting in our code, thus I had to let it reformat a series of files to be compliant with its new exigencies.

Finally `black` was updated and `py39` is fully supported.

## Changes
The following files were changed to support Python 3.9:
  * `.travis.yml`
  * `pyproject.toml`
  * `setup.py`
  * `tox.ini`
  
**All other files were just reformatted to be compliant with the new `black` constraints.**

## Observations
* This PR doesn't change any functionality on IIB, just reformat files and allow the use of Python 3.9
* The latest version of Python is 3.10 however this version is not supported on RHEL8 and RHEL9 beta uses 3.9 as default, thus the inclusion of 3.9 instead of 3.10

Refers to CLOUDDST-11351

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -26,7 +28,7 @@ pytest_command =
 description = black checks [Mandatory]
 skip_install = true
 deps =
-    black==19.10b0
+    black==21.4b0
```
Comment (lipoja) [inline tox.ini:31]: Suggestion - separate PR or at least separate commit for bumping Black version - one can be lost in all those format changes which might not be relevant to the topic "Add support for Python3.9"

**Review by lipoja — CHANGES_REQUESTED** [review]
LGTM, just separate PR/commit for "black magic"

File: tox.ini (hunk 1 above)
Comment (JAVGan) [reply tox.ini:31]: Roger :+1:

Comment (JAVGan) [reply tox.ini:31]: Addressed

**Review by lipoja — APPROVED** [review]
